File: Class-work/OOP/account-polymorphism-app/src/com/monocept/model/Account.java
package com.monocept.model;

public abstract class Account {
	
	protected int accno;
	protected String name;
	protected double balance;
	
	public Account(int accno, String name, double balance)
	{
		this.accno = accno;
		this.name = name;
		this.balance = balance;
	}
	
	public void deptsit(double amount)
	{
		this.balance += amount;
	}
	
	public int getAccno()
	{
		return accno;
	}
	
	public String getName()
	{
		return name;
	}
	
	public double getBalance()
	{
		return balance;
	}

	public abstract void withdraw(double amount);

}

File: NodeJs/module_basics/Account.js
class Account {

    constructor(id,name,balance){
        this.id = id;
        this.name = name;
        this.balance = balance;
    }

    deposit = function(amount) {
        this.balance += amount;
    }

    getId = function() {
        return this.id;
    }

    getName = function() {
        return this.name;
    }

    getBalance = function() {
        return this.balance;
    }
}

module.exports = Account;